_data/other.yml

```
@@ -151,6 +151,16 @@ websites:
     img: dreamstime.png
     twitter: Dreamstime
 
+  - name: Egnyte
+    url: https://www.egnyte.com/
+    img: egnyte.png
+    tfa:
+      - totp
```

Authy push notification would be `proprietary`.
```suggestion
      - totp
      - proprietary
```